app/src/main/java/com/mk/tasky/authentication/login/presentation/LoginViewModel.kt

```diff
@@ -62,7 +70,10 @@ class LoginViewModel @Inject constructor(
         viewModelScope.launch {
             try {
                 repository.login(email, password).onSuccess {
```

Comment:
Was this the try/catch block I commented on previously or another one? It's quite unnecessary

Reply:
Oh nono, I had 2. One for Login and one for Registration
We removed the registration one, but totally forgot about Login

Nice catch!